dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -0,0 +1,364 @@
+                    container.setMaxContentlets(Integer.parseInt(map.get("max_contentlets")));
+                }
+
+                if (map.get("use_div") != null && !map.get("use_div").isEmpty()) {
+                    container.setUseDiv(Boolean.parseBoolean(map.get("use_div")));
+                }
+
+                if (map.get("staticify") != null && !map.get("staticify").isEmpty()) {
+                    container.setStaticify(Boolean.parseBoolean(map.get("staticify")));
+                }
+
+                container.setSortContentletsBy(map.get("sort_contentlets_by"));
+
+                container.setLuceneQuery(map.get("lucene_query"));
+
+                container.setNotes(map.get("notes"));
+
+                container.setIdentifier(map.get("identifier"));
+
+                ret.add(container);
+            }
+        }
+        return ret;
+    }
+
+    public static List<Template> convertDotConnectMapToTemplate(List<Map<String,String>> results)
+            throws ParseException {
+        Template template;
+        List<Template> ret;
+
+        ret = new ArrayList<>();
+
+        if(results != null && !results.isEmpty()){
+            for (Map<String, String> map : results) {
+                template = new Template();
+                template.setInode(map.get("inode"));
+
+                if (map.get("show_on_menu") != null && !map.get("show_on_menu").isEmpty()) {
+                    template.setShowOnMenu(Boolean.parseBoolean(map.get("show_on_menu")));
+                }
+
+                template.setTitle(map.get("title"));
+
+                if (map.get("mod_date") != null && !map.get("mod_date").isEmpty()){
+                    template.setModDate(df.parse(map.get("mod_date")));
+                }
+
+                template.setModUser(map.get("mod_user"));
+
+                if (map.get("sort_order") != null && !map.get("sort_order").isEmpty()){
+                    template.setSortOrder(Integer.parseInt(map.get("sort_order")));
+                }
+
+                template.setFriendlyName(map.get("friendly_name"));
+
+                template.setBody(map.get("body"));
+                template.setHeader(map.get("header"));
+                template.setFooter(map.get("footer"));
+                template.setImage(map.get("image"));
+                template.setIdentifier(map.get("identifier"));
+
+                if (map.get("drawed") != null && !map.get("drawed").isEmpty()) {
+                    template.setDrawed(Boolean.parseBoolean(map.get("drawed")));
+                }
+
+                template.setDrawedBody(map.get("drawed_body"));
+
+                if (map.get("add_container_links") != null && !map.get("add_container_links").isEmpty()){
+                    template.setCountAddContainer(Integer.parseInt(map.get("add_container_links")));
+                }
+
+                if (map.get("containers_added") != null && !map.get("containers_added").isEmpty()){
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "containers_added" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)